src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -13,7 +13,9 @@
      * <p>Это задание необходимо решать с использованием логических операторов,
      * без использования конструкций if-else</p>
      */
+
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return ((a && b) && (!c && !d)) || ((a && c) && (!b && !d)) || ((a && d) && (!c & !b))
+                || ((b && c) && (!a && !d)) || ((b && d) && (!a && !c)) || ((c && d) && (!a && !b));
```

Comment:
try to simplify it with binary operators

Reply:
Simplified